cmd/hack/hack.go

```diff
@@ -1679,7 +1682,7 @@ func testStage5(chaindata string, reset bool) error {
 	log.Info("Stage5", "progress", stage5progress)
 	core.UsePlainStateExecution = true
 	ch := make(chan struct{})
-	stageState := &stagedsync.StageState{Stage: stages.IntermediateHashes, BlockNumber: stage5progress}
+	stageState := &stagedsync.StageState{Stage: stages.IntermediateHashes, BlockNumber: stage4progress}
```

Comment:
It does actually need to start with `stage5progress`, from the place it finished the last time